### DISCUSSION TITLE
ZEPETO Default Project로 프로젝트 생성시 캐릭터 생성 안됨 문제

### DISCUSSION THREAD

**Discussion by Ted-0323** [naverz/zepeto-studio-kor#1202] [5. Q&A]
이전에 사용하던 월드를 패키지로 만든 뒤 ZEPETO Default Project로 프로젝트를 생성한 뒤에 거기에 패키지를 집어넣어 씬을 실행시켰는데도 씬이 실행이 되지 않습니다.

이전 작업때는 그냥 된걸로 알고 있는데, 왜 안될까요...

**Comment by zlmrzlo** [comment]
이건 이미지가 있으면 더 좋을 것 같네요. 어느 오류가 발생되는지 확인하면 필요한 정보를 과거에 한 번 경험한 분들이 답변해주실 것 같아요.
